src/Icons/src/Command/SearchIconCommand.php

```diff
@@ -107,14 +109,20 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         if (1 < \count($iconSets) && $prefix !== $iconSet['prefix']) {
             $choices = array_combine(array_keys($iconSets), array_column($iconSets, 'name'));
             $choice = $io->choice('Select an icon set', array_values($choices));
-            if (!$choice || false === $prefix = array_search($choice, $choices, true)) {
+            if (!$choice || false === $prefix = array_search($choice, $choices, true) || !is_string($prefix)) {
```

Comment:
Can be simplified to that, right?
```suggestion
            if (!$choice || !is_string($prefix = array_search($choice, $choices, true))) {
```